README.md

```diff
@@ -1,13 +1,53 @@
-## Getting Started
 
-First, run the development server:
+# Cron-AI
+
+Introducing the AI Cron Job Code Generator - a cutting-edge app that uses advanced AI algorithms to generate optimized cron job codes in seconds. 
```

Comment:
can we update this sentence in a way so that it gives proper credit to OpenAI GPT-3 API (specifically, text-davinci-003)